src/pages/extensions/index.tsx

```diff
@@ -132,6 +133,11 @@ const ExtensionsPage = () => {
         downloadLink="https://github.com/Testaustime/testaustime-micro"
         sourceCodeLink="https://github.com/Testaustime/testaustime-micro"
         text={t("extensions.micro")}
+      /><ExtensionBlock
```

Comment:
Lets place these on separate lines
```suggestion
      />
      <ExtensionBlock
```